common/lib/xmodule/xmodule/lti_module.py

```diff
@@ -918,11 +918,11 @@ def is_past_due(self):
             close_date = due_date + self.graceperiod  # pylint: disable=no-member
         else:
             close_date = due_date
-<<<<<<< HEAD
-        return (close_date is not None and datetime.datetime.now(UTC()) > close_date)
-=======
+        # TODO:FUNK <<<<<<< HEAD
+        # return (close_date is not None and datetime.datetime.now(UTC()) > close_date)
+        # TODO:FUNK =======
         return close_date is not None and datetime.datetime.now(UTC()) > close_date
->>>>>>> 00b75f0119b981641833240be214ef2076329747
+        # TODO:FUNK >>>>>>> 00b75f0119b981641833240be214ef2076329747
```

Comment:
Weird, i blamed upstream and it was my commit, but I didn't have the parens. I think keeping parens is cleaner, or should we just stick with upstream to avoid conflicts later?